Flashcards.DouglasFir/Flashcards/App.config

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+	<appSettings>
+		<!--' Database Constants | Database, Table, and View names '-->
+		<add key="DbName" value="FLASHCARDS" />
+		<add key="TblStackName" value="tb_Stacks" />
+		<add key="TblFlashCardsName" value="tb_FlashCards" />
+		<add key="TblStudySessionsName" value="tb_StudySessions" />
+		<add key="VwFlashCardsName" value="vw_FlashCardsDisplay" />
+		<add key="VwStudySessionsName" value="vw_StudySessionsWithStacks" />
+		<add key="VwFlashCardsRenumberedName" value="vw_FlashCardsRenumbered" />
```

Comment:
Please use readable names. I can guess tbl means table, but I don't know what vw stands for. Have a look at point 4 of the [Code Conventions](https://thecsharpacademy.com/article/58/code-conventions).

Reply:
Corrected on point 4. I was mixing conventions between sql and C#, but I do see the value in verbose names.

Reply:
@rankdjr I don't see a new comment yet, correct?

Reply:
@antonderegt correct. had a busy few weeks, so just getting around to finishing these changes